content/docs/console/continuous-delivery/github-actions.md

```diff
@@ -159,31 +167,50 @@ To edit your secrets, scroll down and select the first Pulumi action, and click
 
 ![Editor Edit](/images/docs/reference/gh-actions-editor-configure.png)
 
-This will open the action editor on the right-hand side, in which you'll enter your various credentials (for Pulumi and
-your cloud). After configuring these, you should see the green secrets with lock icons show up for your actions:
+This will open the action editor on the right-hand side, in which you'll enter your
+various credentials (for Pulumi and your cloud). After configuring these, you should see
+the green secrets with lock icons show up for your actions:
 
 ![Editor Secrets](/images/docs/reference/gh-actions-editor-secrets.png)
 
+# Try It Out!
+
+To try things out, simply create a Pull Request or commit, and you will see these new
+actions showing up in the usual GitHub Checks dialog, with a green checkmark if everything
+went as planned:
+
+![Action Checks](/images/docs/reference/gh-actions-checks.png)
+
+Click the Logs pane to see the full output of the Pulumi CLI, along with the URL of your
+deployment on the Pulumi Cloud Console with more details:
+
+![Action Logs](/images/docs/reference/gh-actions-logs.png)
+
+For even better Pull Request integration, make sure to also [install our GitHub App]({{< relref "github.md" >}})!
+
+![Action Pull Requests](/images/docs/reference/gh-actions-prs.png)
+
 # Pull Request Flow
 
-If you are using Pulumi's GitHub Actions to preview infrastructure changes from Pull Requests, you may want to
-have Pulumi comment on those PRs so that you don't need to look at the specific update logs to see if there
-were any changes.
+If you are using Pulumi's GitHub Actions to preview infrastructure changes from Pull
+Requests, you may want to have Pulumi comment on those PRs so that you don't need to look
+at the specific update logs to see if there were any changes.
 
-There are two ways to do this: using the Pulumi GitHub App (recommended), or configuring the GitHub Actions
-container directly.
+There are two ways to do this: using the Pulumi GitHub App (recommended), or configuring
+the GitHub Actions container directly.
 
 ## Pulumi GitHub App
 
-The [Pulumi GitHub App]({{< relref "github.md" >}}) is something you install into your GitHub
-organization, that will allow the Pulumi service to leave comments on Pull Requests. (It will not have access
-to your source code.)
+The [Pulumi GitHub App]({{< relref "github.md" >}}) is something you install into your
+GitHub organization. It allows the Pulumi service to leave comments on Pull Requests. (It
+does not have access to your source code.)
 
-Once the Pulumi GitHub App is installed, when your GitHub Actions run Pulumi, a summary of any resource changes
-is left on the Pull Request. As well as links to the Pulumi Cloud Console for more detailed information.
+Once the Pulumi GitHub App is installed, when your GitHub Actions run Pulumi, a summary of
```

Comment:
Suggest changing to:

Once the Pulumi GitHub App is installed, a summary of resource changes---including links to the Pulumi Cloud Console for more detailed information---will be left on the Pull Request when your GitHub Actions run Pulumi.